raycicmd/make.go

```diff
@@ -70,14 +70,18 @@ func parsePipelineFile(file string) (*pipelineGroup, error) {
 	}
 
 	g.filename = file
-	g.name = stripRayCIYamlSuffix(filepath.Base(file))
+	if g.SortKey != "" {
+		g.sortKey = g.SortKey
+	} else {
+		g.sortKey = stripRayCIYamlSuffix(filepath.Base(file))
```

Comment:
since it's a suffix do we need to trim it for sorting?

Reply:
it probably won't affect the sorting result in many cases, but it should be removed as it is going to be compared to strings without the suffix.